otus/freebsd/src/sys/dev/athp/if_athp_core.c

```diff
@@ -1690,18 +1690,41 @@ ath10k_core_probe_fw(struct ath10k *ar)
 	return ret;
 }
 
+static void
+clean_ath10k_core_probe_fw(struct ath10k * ar) {
+	//This function has been made to do cleanup to prevent memory leaks if any exist.
+	//First step tell bmi done has not been sent so it will re-setup bmi.
+	ar->bmi.done_sent = false;
+}
+
+static int
+attempt_ath10k_core_probe_fw(struct ath10k *ar, int anum) {
+	//anum is the number of attempts that have been tried.
```

Comment:
Put the comments above the function declaration!

Reply:
Got it!